.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .DS_Store*
 .hubot_history
+*.sw[o|p]
```

Comment:
Cool. In that case, it would be great if you could remove this commit from the PR.